.github/vale/dicts/aiven.dic

```diff
@@ -2,6 +2,7 @@
 ACL
 ACLs
 Aiven
+Aiven Console
```

Comment:
I don't think this is necessary: Aiven is already present and Console is a correct word